addons/web/static/src/views/calendar/calendar_common/calendar_common_week_column.js

```diff
@@ -0,0 +1,18 @@
+/** @odoo-module **/
+
+export function makeWeekColumn({ el, showWeek, weekColumn, weekText }) {
+    const firstRows = el.querySelectorAll(".fc-col-header-cell:nth-child(1), .fc-day:nth-child(1)");
+    [].map.call(firstRows, (element) => {
```

Comment:
Wouldn't a `for ... of ...` loop be easier to read ?
```suggestion
    for(const element of firstRows) {
```